build.gradle

```diff
@@ -49,7 +49,7 @@ jastadd {
 	parser.name = 'JavaParser'
 }
 
-generateJava << {
+generateJava {
```

Comment:
I think the right thing to do here is to use

    generateJava {
        doLast { ... }